lm_eval/evaluator.py

```diff
@@ -550,7 +595,10 @@ def evaluate(
                                     + metric_score * current_size
                                 ) / (total_size + current_size)
                                 # $$s_z^2 = \frac{(n-1) s_x^2 + (m-1) s_y^2}{n+m-1} + \frac{nm(\bar x - \bar y)^2}{(n+m)(n+m-1)}.$$
-                                if var_score == "N/A" or results[group][stderr] == "N/A":
+                                if (
```

Comment:
will have to merge into main, as the variance/ stderr calc above has changed (this shouldn't affect your PR though, just merge conflicts with these formatting changes)

Reply:
Roger roger, lmk if anything needs changing on my end